test/apicoverage-webhook/README.md

```diff
@@ -0,0 +1,18 @@
+## Serving Webhook based API Coverage Image
+
+This directory contains the http.Server image used in Webhook based API
```

Comment:
"HTTP server", here and everywhere? I see no module/class/object named this way.

Reply:
Fixed.

Reply:
Yes was addressing all the comments before pushing the changes. Will do in few mins.